grumpy-rainbows/BangazonAPI
# Bj computer controller

*Description*

Creating controllers(GET, POST, PUT, and DELETE) for Computer class. Also creating tests for each of the controllers as well as tests using each of the query parameters"

Fixes Issue # Issue #9

*Type of change*

Please delete options that are not relevant.
• [x] New feature (non-breaking change which adds functionality)


*Testing Instructions*

1. `git fetch --all`
2. `git checkout BJComputerController`
3. Run all tests in TestComputer and verify that all tests work. 

*Checklist:*

• [x] My code follows the style guidelines of this project
• [x] I have performed a self-review of my own code
• [x] I have commented my code, particularly in hard-to-understand areas
• [x] I have made corresponding changes to the documentation
• [x] My changes generate no new warnings
• [x] I have added test instructions that prove my fix is effective or that my feature works
